nipype/interfaces/io.py

```diff
@@ -2810,8 +2812,10 @@ def __init__(self, infields=None, **kwargs):
 
         if not isdefined(self.inputs.output_query):
             self.inputs.output_query = {
-                "func": {"modality": "func", 'extensions': ['nii', '.nii.gz']},
-                "anat": {"modality": "anat", 'extensions': ['nii', '.nii.gz']},
+                "bold": {"datatype": "func", "suffix": "bold",
+                         "extensions": ["nii", ".nii.gz"]},
+                "T1w": {"datatype": "anat", "suffix": "T1w",
+                         "extensions": ["nii", ".nii.gz"]},
```

Comment:
Not a problem for this PR, but we may want to think about making it a much more generally usable default.

Reply:
Sure. Such as? you mean returning more things?

Reply:
Yeah. Ideally something that will contain most, if not all, files, and can just be queried sensibly. I feel like specifying the query is more of a power-user move, so making sure there's a reasonable way to ask for the files most people will want would be good.

Reply:
That sounds good to me, although I worry about about performance if a bunch of unnecessary queries are made. Although the indexing itself is probably the slowest part, so it should probably be fine.

Reply:
While we're at it, other fields you'd want? `brain_mask` is another one I can think of for fmri. I would like to hear from ppl doing preprocessing and and DWI as I'm unfamiliar with what is typically expected.